packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/__tests__/quanticSearchBoxSuggestionsList.test.js

```diff
@@ -131,6 +120,21 @@ const buildMixedOptions = (suggestions, recentQueries) => {
   ];
 };
 
+const functionsMocks = {
+  highlightString: jest.fn((payload) => payload.content),
```

Comment:
why are we no longer testing for the highlight? I see you marked the utils for the formatting as deprecated but not sure I get why